rosdep/python.yaml

```
@@ -7773,6 +7773,13 @@ python3-sklearn:
     pip:
       packages: [scikit-learn]
   ubuntu: [python3-sklearn]
+python3-smbus:
+  arch: [i2c-tools]
+  debian: [python3-smbus]
+  fedora: [i2c-tools]
```

Though provided by the `i2c-tools` source package, the actual python bindings are provided in the `python3-i2c-tools` subpackage: https://src.fedoraproject.org/rpms/i2c-tools/blob/rawhide/f/i2c-tools.spec#_32

```suggestion
  fedora: [python3-i2c-tools]
```